backend/app/crud/assistants.py

```
@@ -1,32 +1,57 @@
 import logging
 
-from typing import Optional
+from typing import Optional, Set
```

Ruff (0.12.2)

`typing.Set` is deprecated, use `set` instead

(UP035)